src/ceres_infer/models.py

```diff
@@ -740,13 +740,17 @@ def model_infer_ens_custom(data, dm_model, feat_labels, target_name, df_res, y_c
     dm_model.model.set_params(max_depth=7)
     feat_selector = BorutaPy(dm_model.model, n_estimators='auto', verbose=0)
     feat_selector.fit(x_tr, y_train)
-
+    
     feat_names_sel = feat_names_sel[feat_selector.support_]
     if len(feat_names_sel) < 1: return df_res, None
     x_tr = feat_selector.transform(x_tr)
     x_te = feat_selector.transform(x_te)
     sf = _featSelect_base()
     sf.importance_sel = pd.DataFrame(feat_names_sel.copy())
+    
+    # if don't set quantile, the column name will still be name not feature
```

Comment:
When does this happen? was this affecting our previous runs/results?

Reply:
Hi Bo, below is my intepretation and why I added the code:
The original column name of feature is 'name'
https://github.com/pritchardlabatpsu/cnp_dev/blob/5b581a6b5dd950ad64c2dd580f706cf45d04ac45/src/ceres_infer/models.py#L723
But during iterative selection, the column name will be updated to 'feature'
https://github.com/pritchardlabatpsu/cnp_dev/blob/5b581a6b5dd950ad64c2dd580f706cf45d04ac45/src/ceres_infer/models.py#L726-L727
When we run model_infer_ens_custom, the value of sf_iterThreshold is empty in pipeline_params. Because it didn't undergo iterative selection, the column name is still 'name'. That's why I updated it manually to 'feature'
https://github.com/pritchardlabatpsu/cnp_dev/blob/5b581a6b5dd950ad64c2dd580f706cf45d04ac45/src/ceres_infer/models.py#L725-L727
otherwise we can not fit the model
https://github.com/pritchardlabatpsu/cnp_dev/blob/5b581a6b5dd950ad64c2dd580f706cf45d04ac45/src/ceres_infer/session.py#L650

Reply:
When I run run04_reg_200_rfboruta I got the same error, did you get it before? If not then this is weird...

Reply:
The original runs was based on a method called `model_infer_ens`, which was then generalized to custom. The runs were already finished, and it was modified to `model_infer_ens_custom` post-run just in the notebooks. I don't think changing name to feature is right, as `name` is a column that just contains the gene names, while `feature` is values of *gene [source]* format.

From a old commit: 
https://github.com/pritchardlabatpsu/cnp_dev/blob/ab68be019a6e292175a5cb3c4207c028e18fc1bb/src/ceres_infer/models.py#L675-L717

Reply:
The alternative is to add in a dummy [0], which was used in previous `model_infer_ens`, 

```
    # set up feat_names_sel values and copy train sets, for use in iterative selection next
    x_tr = x_train.copy()
    x_te = x_test.copy()
    feat_names_sel = feat_labels.name

    if len(sf_iterThresholds) < 1:
        # if not threshold are set. Set a dummy threshold, so we can get the feat_names_sel structure
        sf_iterThresholds = [0]
```

Reply:
Please see my latest commit. It alternatively changes where to update the feature to here.

https://github.com/pritchardlabatpsu/cnp_dev/blob/fd512e72ffda8203144b0844947a242b66988f64/src/ceres_infer/models.py#L723-L724

Reply:
If this looks good, I have no further comments. You can resolve this and merge whenever you're ready.